packages/icons/icons.json

```
@@ -3075,644 +3075,644 @@
         "group": "data",
         "codepoint": 61818
     },
     {
         "displayName": "Crown",
         "iconName": "crown",
-        "tags": "crown, hat, role, level, permission, king",
+        "tags": "hat, role, level, permission, king, owner",
         "group": "miscellaneous",
         "codepoint": 61802
     },
     {
         "displayName": "Globe: network",
         "iconName": "globe-network",
-        "tags": "planet, earth, map, world, www, website",
+        "tags": "planet, earth, map, world, internet, website",
         "group": "miscellaneous",
         "codepoint": 61906
     },
     {
         "displayName": "Snowflake",
         "iconName": "snowflake",
-        "tags": "weather, winter, freeze, ice",
+        "tags": "weather, winter, freeze, ice, snow",
         "group": "miscellaneous",
         "codepoint": 62143
     },
     {
         "displayName": "Tree",
         "iconName": "tree",
-        "tags": "forest, wood, landmark",
+        "tags": "forest, wood, landmark, conifer",
         "group": "miscellaneous",
         "codepoint": 62203
     },
     {
         "displayName": "Notifications: updated",
         "iconName": "notifications-updated",
-        "tags": "notifications, bell, alarm, notice, warning",
+        "tags": "bell, alarm, alert, notice, warning, resolve, approve, confirm, check, clear, read, configured, acknowledge, ack",
         "group": "interface",
         "codepoint": 62045
     },
     {
         "displayName": "List: columns",
         "iconName": "list-columns",
-        "tags": "layout, arrangement, four lines, table",
+        "tags": "layout, arrangement, four lines, table, two, article, data, grid",
         "group": "table",
         "codepoint": 61998
     },
     {
         "displayName": "Flow: linear",
         "iconName": "flow-linear",
-        "tags": "workflow, edge, node, step",
+        "tags": "workflow, edge, node, step, event, timeline, directed edge, right, advance, route, valve",
         "group": "data",
         "codepoint": 61876
     },
     {
         "displayName": "Flow: branch",
         "iconName": "flow-branch",
-        "tags": "workflow, edge, node, fork",
+        "tags": "workflow, edge, node, fork, arrows, split, two-way, decision point, split path",
         "group": "data",
         "codepoint": 61874
     },
     {
         "displayName": "Flow: review",
         "iconName": "flow-review",
-        "tags": "workflow, edge, node, reverse",
+        "tags": "workflow, edge, node, reverse, expand, widen, tie fighter, split",
         "group": "data",
         "codepoint": 61878
     },
     {
         "displayName": "Flow: review branch",
         "iconName": "flow-review-branch",
-        "tags": "workflow, edge, node, fork",
+        "tags": "workflow, edge, node, fork, locate, axis, graph, outward, away, location, relationship, links, unknown, geolocate, map, grappling hook",
```

there are a decent number of "locate" / "location" icons already, so I don't think we need to conflate these with map/location terminology

```suggestion
        "tags": "workflow, edge, node, fork, axis, graph, outward, away, location, relationship, links, unknown, grappling hook",
```